spec/support/jasmine-browser.json

```diff
@@ -1,16 +1,20 @@
 {
-  "srcDir": "public/assets/frontend",
+  "srcDir": ".",
   "srcFiles": [
-    "test-dependencies-*.js",
-    "application-*.js",
-    "views/travel-advice-*.js"
+    "public/assets/frontend/test-dependencies-*.js",
+    "public/assets/frontend/dependencies-*.js",
+    ".nyc_output/assets/**/*.js"
   ],
   "specDir": "spec/javascripts",
   "specFiles": [
     "**/*[sS]pec.js"
   ],
   "helpers": [
+    "helpers/*.mjs",
     "vendor/*.js"
   ],
-  "browser": "headlessChrome"
+  "browser": "headlessChrome",
+  "reporters": [
+    "./spec/support/reporters/test-coverage.mjs"
```

Comment:
Could we update the file name to include the name of the reporter, for example: `nyc-test-converage.js`

Reply:
Done